internal/pkg/show/config/config.go

```diff
@@ -6,14 +6,16 @@ import (
 	"github.com/spf13/viper"
 )
 
+var templates = map[string]string{
+	"quickstart": QuickStart,
+	"gitops":     GitOps,
+}
+
 //go:generate go run gen_embed_var.go
 func Show() error {
-	template := viper.GetString("template")
-	if template == "quickstart" {
-		fmt.Println(QuickStart)
-		return nil
-	} else if template == "gitops" {
-		fmt.Println(GitOps)
+	// at first, check is template parameter is set
+	if tmpl, ok := templates[viper.GetString("template")]; ok {
```

Comment:
What if a user type: `dtm show config -t abcd`?

It is possible that some exception input checks are needed here to inform the user what parameters are supported when the user enters an incorrect parameter.